### PR TITLE
Make a high-level outline for the toolkit

### PR DESCRIPTION
**Pull request by kfogel**

### CODE REVIEW

File: outline.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+* Introduction
+  - A Brief History of Open
```
Comment (lacabra) [inline outline.md:2]: Is this a typo? or it is meant to be like this? Intuitively I would expect "Open something"

File: outline.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  - Types of Digital Public Goods
```
Comment (lacabra) [inline outline.md:3]: For the sake of consistency, we have been referring to (in alphabetical order):
- Open AI models
- Open content
- Open data
- Open standards
- Open-source software

Unless you include a strong rationale for changing these categories, we will ask to use the existing nomenclature.

**Review by lacabra — CHANGES_REQUESTED** [review]

File: outline.md (hunk 1 above)
Comment (kfogel) [reply outline.md]: It's not a typo, but the fact that you hesitated on reading it indicates that maybe we should change the wording.

For now I've changed it to "A Brief History of Open Licensing".  I don't like focusing on licensing like that (as opposed to focusing on collaboration), but we're not presenting a brief history of open collaboration throughout history -- that would go back many thousands of years! -- but rather a brief history of the digital collaboration that arose from open licensing.

We may think of something better, and suggestions welcome!  This isn't written in stone, naturally.

File: outline.md (hunk 2 above)
Comment (kfogel) [reply outline.md:3]: Done and pushed to branch; thank you.

File: outline.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    + [Qalipso's OMM](https://en.wikipedia.org/wiki/OpenSource_Maturity_Model)
+    + Open Source Skills Stack
+      - Economic / Business
+      - Political / Strategic
+      - Social
+      - Licensing
+      - Technical
+  - Climbing The Readiness Curve
+
+* Procurement
```
Comment (imahgoub) [inline outline.md:42]: Can we include here minimum requirements for procurement processes to be DPG enabling?

Comment (imahgoub) [inline outline.md:42]:
```suggestion
* Procurement

Can we include here minimum requirements for procurement processes to be DPG enabling?


```

File: outline.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  - Modular Contracting
+  - IP Contract Terms
+  - Open Source Quality Assurance
+  - Staffing
+  - Open Source Solicitation
+
+* Adoptability Assessment
```
Comment (imahgoub) [inline outline.md:49]:
```suggestion
* Adoptability Assessment
```

Landscape analysis/mapping of technical capacity and gaps

File: outline.md (hunk 1 above)
Comment (imahgoub) [reply outline.md]: I would suggest for the scope of the toolkit to narrow this down to a brief history of open licensing in the public sector specifically.

File: outline.md (hunk 4 above)
Comment (kfogel) [reply outline.md:49]: @imahgoub Is the "Navigating Existing DPGs" item about searching and findability -- i.e., given a certain need, this item is about how to locate a DPG or DPGs that might meet that need?

Comment (kfogel) [reply outline.md:49]: (I went with the assumption above, and put that new item under Procurement rather than Adoptability Assessment.)

**Review by imahgoub — APPROVED** [review]

**Review by imahgoub — APPROVED** [review]

**Review by lacabra — APPROVED** [review]

**Pull request merged**